service/src/main/resources/application.yaml

```diff
@@ -106,6 +106,6 @@ fpl:
   local_authority_code_to_name:
     mapping: 'SA=>Swansea City Council;HN=>London Borough Hillingdon;PCC=>Portsmouth City Council;STF=>Staffordshire County Council;SCC=>Southampton City Council'
   local_authority_code_to_hmcts_court:
-    mapping: 'SA=>Swansea Family Court:FamilyPublicLaw+sa@gmail.com;HN=>Portsmouth Combined Court:FamilyPublicLaw+hn@gmail.com'
+    mapping: 'SA=>Family Court sitting at Swansea:FamilyPublicLaw+sa@gmail.com;HN=>Family Court sitting at Portsmouth:FamilyPublicLaw+hn@gmail.com'
```

Comment:
does this file actually need changing?

Reply:
yes - to reflect the correct court name format on locally generated documents
I have also now updated the key vault secret for aat and sent the correct mapping to @akomisarek so he can update prod (should only be updated on prod after story relating to email notifications is complete otherwise court name on email will be wrong)